one_fm/api/doc_methods/shift_request.py

```diff
@@ -183,6 +203,16 @@ def update_shift_assignment(shift_assignemnt,shift_request):
         assignment_doc.db_set("end_datetime" , end_datetime)
     if shift_request.operations_role:
         assignment_doc.db_set("operations_role" , shift_request.operations_role)
+    # employee_checkin_update = frappe.db.sql(f"""UPDATE `tabEmployee Checkin` SET employee_is_replaced=1 WHERE shift_assignment='{shift_assignemnt}'""")
+
+def replace_shift_assignment(shift_assignemnt, doc):

```

Comment:
This method have no body

Reply:
The body for this method was adding the following story, which is sub story of this Story.